### PR TITLE
Remove pickles older than 12 hrs

### PR DESCRIPTION
**Pull request by andreimarinescu**

### CODE REVIEW

**Comment by schnie** [comment]
@andreimarinescu wondering if we could move `dag_pickle` cleanup to it's own file.  It''s not really related directly to `cloud_dags`, and we'll be adding more dag files in soon.  What do you think?

**Comment by andreimarinescu** [comment]
Updated the PR as discussed. The DAG import and default_args dictionary seem to be needed, as the file wouldn't be run by airflow before adding these. I've tested this on my own DC/OS cluster, let me know if you have any issues. 

A scheduler log showing this in action:

    [2016-10-25 14:37:03,358] {models.py:238} DEBUG - Importing /airflow/dags/clean_pickles.py
    Cleaning old pickles
    Pickles clean
    [2016-10-25 14:37:03,395] {models.py:238} DEBUG - Importing /airflow/dags/cloud_dags.py
    2016-10-25 14:37:04,396 INFO success: scheduler entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
    Connecting to mongodb.
    Querying for cloud workflows.
    Found 2 workflows.
    Building DAG: 1_dev_astronomer_mongo_to_redshift_applications__etl__57f511b52be55e58cf83ac30.
    Building DAG: 2_dev_astronomer_mongo_to_redshift_applications__etl__57ffada82be55e58cf83ac6f.
    Finished exporting ETL DAG's.
    [2016-10-25 14:37:04,406] {models.py:367} DEBUG - Loaded DAG <DAG: 2_dev_astronomer_mongo_to_redshift_applications__etl__57ffada82be55e58cf83ac6f>
    [2016-10-25 14:37:04,409] {models.py:367} DEBUG - Loaded DAG <DAG: 1_dev_astronomer_mongo_to_redshift_applications__etl__57f511b52be55e58cf83ac30>
    [2016-10-25 14:37:04,413] {models.py:367} DEBUG - Loaded DAG <DAG: 2_dev_astronomer_mongo_to_redshift_applications__etl__57ffada82be55e58cf83ac6f>
    [2016-10-25 14:37:04,413] {models.py:238} DEBUG - Importing /airflow/dags/webhooks_dags.py

File: airflow/dags/clean_pickles.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from airflow import DAG, models, settings
+from datetime import datetime, timedelta
+from airflow.operators.bash_operator import BashOperator
+
+
+default_args = {
```
Comment (schnie) [inline airflow/dags/clean_pickles.py]: @andreimarinescu We shouldn't need these for this file should we?

File: airflow/dags/clean_pickles.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from airflow import DAG, models, settings
+from datetime import datetime, timedelta
+from airflow.operators.bash_operator import BashOperator
```
Comment (schnie) [inline airflow/dags/clean_pickles.py]: @andreimarinescu We can probably pull this import right?

**Comment by schnie** [comment]
@andreimarinescu I missed your comment about the default_args, sorry about that!  Any idea why that is?

**Pull request merged**